src/backer/backering.py

```diff
@@ -310,6 +310,22 @@ def on_post(self, req, rep):
 
         cr = httping.parseCesrHttpRequest(req=req)
         serder = eventing.Serder(ked=cr.payload, kind=eventing.Serials.json)
+
+        backer_identifier_ked = self.hab.kevers[self.hab.pre].serder.ked
+        # Confirm registry backer
+        if "RB" not in backer_identifier_ked["c"]:
+            raise falcon.HTTPBadRequest(falcon.HTTP_400)
+
+        for item in backer_identifier_ked["a"]:
```

Comment:
There might be multiple entries in `a`. We actually just need to search until we find one that matches correctly. If we find it, break the loop and continue normally. If we don't find it, raise a 400.

Reply:
updated
ref here: 9341a6c17b6fac709ccee64f16870fdf8bd0b81b